### ISSUE TITLE
jetty client hides root cause (handshakeexception) when contacting HTTP2 server with wrong certificate

### ISSUE THREAD

**Issue by janssk1** [eclipse/jetty.project#1295]
Jetty async client version: 9.3.15.v20161220 

I'm trying to access APNS (HTTP2 server) with jetty. 
All works ok with a good certificate, but if a wrong (expired) on is used, the client just raises a generic 'EofException' , hiding the real root cause. Sometimes, the client does return the correct 'SslHandShakeException', so i suspect there is a race condition somewhere.

The debug log below shows the handhakeerror that manifest itself as an Eof. 

```
09:54:44,305 DEBUG [SslConnection] SslConnection@531b92c7{NEED_WRAP,eio=-1/-1,di=-1} -> HTTP2ClientConnection@7213547c[DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,Open,in,out,-,C,237/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,Open,in,out,-,-,5/0,SslConnection@531b92c7}{io=1/0,kio=1,kro=1}] flush b[15]=00000604000000000000040000FFFF
09:54:44,306 DEBUG [SslConnection] SslConnection@531b92c7{NEED_UNWRAP,eio=-1/31,di=-1} -> HTTP2ClientConnection@7213547c[DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,Open,in,OSHUT,-,C,238/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,Open,in,out,-,-,6/0,SslConnection@531b92c7}{io=1/0,kio=1,kro=1}] wrap Status = CLOSED HandshakeStatus = NEED_UNWRAP bytesConsumed = 0 bytesProduced = 31
09:54:44,307 DEBUG [ChannelEndPoint] flushed 31 SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,Open,in,out,-,-,7/0,SslConnection@531b92c7}{io=1/0,kio=1,kro=1}
09:54:44,307 DEBUG [ChannelEndPoint] oshut SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,Open,in,out,-,-,0/0,SslConnection@531b92c7}{io=1/0,kio=1,kro=1}
09:54:44,308 DEBUG [SslConnection] SslConnection@531b92c7{NEED_UNWRAP,eio=-1/-1,di=-1} -> HTTP2ClientConnection@7213547c[DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,Open,in,OSHUT,-,C,240/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,Open,in,OSHUT,FI,-,1/0,SslConnection@531b92c7}{io=1/0,kio=1,kro=1}] shutdownOutput: oshut=true, ishut=false
09:54:44,308 DEBUG [FillInterest] FillInterest@21468e50{true,AC.ReadCB@531b92c7{SslConnection@531b92c7{NEED_UNWRAP,eio=-1/-1,di=-1} -> HTTP2ClientConnection@7213547c[DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,Open,in,OSHUT,-,C,239/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,Open,in,OSHUT,FI,-,0/0,SslConnection@531b92c7}{io=1/0,kio=1,kro=1}]}} register AC.ReadCB@531b92c7{SslConnection@531b92c7{NEED_UNWRAP,eio=-1/-1,di=-1} -> HTTP2ClientConnection@7213547c[DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,Open,in,OSHUT,-,C,240/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,Open,in,OSHUT,FI,-,1/0,SslConnection@531b92c7}{io=1/0,kio=1,kro=1}]}
09:54:44,309 DEBUG [WriteFlusher] ignored: WriteFlusher@1b0323bb{IDLE} java.nio.channels.ClosedChannelException
09:54:44,313 DEBUG [ChannelEndPoint] close SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,CLOSED,in,OSHUT,-,-,6/0,SslConnection@531b92c7}{io=1/0,kio=1,kro=1}
09:54:44,314 DEBUG [FillInterest] FillInterest@21468e50{true,AC.ReadCB@531b92c7{SslConnection@531b92c7{NEED_UNWRAP,eio=-1/-1,di=-1} -> HTTP2ClientConnection@7213547c[DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,CLOSED,in,OSHUT,-,C,245/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,CLOSED,in,OSHUT,-,-,6/0,SslConnection@531b92c7}{io=1/0,kio=1,kro=1}]}} register AC.ReadCB@531b92c7{SslConnection@531b92c7{NEED_UNWRAP,eio=-1/-1,di=-1} -> HTTP2ClientConnection@7213547c[DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,CLOSED,in,OSHUT,-,C,245/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,CLOSED,in,OSHUT,-,-,6/0,SslConnection@531b92c7}{io=1/0,kio=1,kro=1}]}
09:54:44,315 DEBUG [ManagedSelector] Queued change org.eclipse.jetty.io.ManagedSelector$2@be8a41f on org.eclipse.jetty.io.ManagedSelector@57d5872c id=0 keys=1 selected=1
09:54:44,315 DEBUG [SelectChannelEndPoint] changeInterests p=true 0->1 for SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,CLOSED,ISHUT,OSHUT,-,-,8/0,SslConnection@531b92c7}{io=1/1,kio=-1,kro=-1}
09:54:44,316 DEBUG [WriteFlusher] failed: WriteFlusher@40f03b3b{COMPLETING} org.eclipse.jetty.io.EofException
09:54:44,316 DEBUG [WriteFlusher] failed: WriteFlusher@40f03b3b{COMPLETING} javax.net.ssl.SSLHandshakeException: Received fatal alert: internal_error
09:54:44,316 DEBUG [WriteFlusher] update WriteFlusher@40f03b3b{FAILED}:COMPLETING-->FAILED
09:54:44,317 DEBUG [WriteFlusher] update WriteFlusher@40f03b3b{FAILED}:COMPLETING!->FAILED
09:54:44,317 DEBUG [WriteFlusher] ignored: WriteFlusher@40f03b3b{FAILED} javax.net.ssl.SSLHandshakeException: Received fatal alert: internal_error
09:54:44,319 DEBUG [HTTP2Connection] Could not read from DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,CLOSED,in,OSHUT,-,F,249/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,CLOSED,ISHUT,OSHUT,-,-,10/0,SslConnection@531b92c7}{io=1/1,kio=-1,kro=-1}
javax.net.ssl.SSLHandshakeException: Received fatal alert: internal_error
	at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.fill(SslConnection.java:715)
	at org.eclipse.jetty.http2.HTTP2Connection.fill(HTTP2Connection.java:119)
	at org.eclipse.jetty.http2.HTTP2Connection.access$600(HTTP2Connection.java:38)
	at org.eclipse.jetty.http2.HTTP2Connection$HTTP2Producer.produce(HTTP2Connection.java:211)
	at org.eclipse.jetty.util.thread.strategy.ProduceConsume.execute(ProduceConsume.java:69)
	at org.eclipse.jetty.http2.HTTP2Connection.onFillable(HTTP2Connection.java:110)
	at org.eclipse.jetty.http2.HTTP2Connection$FillCallback.succeeded(HTTP2Connection.java:251)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:110)
	at org.eclipse.jetty.io.ssl.SslConnection.onFillable(SslConnection.java:221)
	at org.eclipse.jetty.io.ssl.SslConnection$3.succeeded(SslConnection.java:116)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:110)
	at org.eclipse.jetty.io.SelectChannelEndPoint$2.run(SelectChannelEndPoint.java:93)
	at org.eclipse.jetty.io.SelectChannelEndPoint.onSelected(SelectChannelEndPoint.java:186)
	at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.processSelected(ManagedSelector.java:283)
	at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:181)
	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.produceExecuteConsume(ExecuteProduceConsume.java:169)
	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.produceConsume(ExecuteProduceConsume.java:145)
	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.run(ExecuteProduceConsume.java:136)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:671)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:589)
	at java.lang.Thread.run(Thread.java:745)
Caused by: javax.net.ssl.SSLException: Received fatal alert: internal_error
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:208)
	at sun.security.ssl.SSLEngineImpl.fatal(SSLEngineImpl.java:1666)
	at sun.security.ssl.SSLEngineImpl.fatal(SSLEngineImpl.java:1634)
	at sun.security.ssl.SSLEngineImpl.recvAlert(SSLEngineImpl.java:1800)
	at sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:1083)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(SSLEngineImpl.java:907)
	at sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:781)
	at javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:624)
	at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.fill(SslConnection.java:553)
	... 20 more
09:54:44,319 DEBUG [WriteFlusher] completeWrite exception
org.eclipse.jetty.io.EofException
	at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.flush(SslConnection.java:786)
	at org.eclipse.jetty.io.WriteFlusher.flush(WriteFlusher.java:420)
	at org.eclipse.jetty.io.WriteFlusher.completeWrite(WriteFlusher.java:375)
	at org.eclipse.jetty.io.ssl.SslConnection$1.run(SslConnection.java:99)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:671)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:589)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.nio.channels.ClosedChannelException
	... 7 more
09:54:44,326 DEBUG [HTTP2Connection] Filled -1 bytes
09:54:44,327 DEBUG [WriteFlusher] update WriteFlusher@40f03b3b{FAILED}:COMPLETING!->IDLE
09:54:44,327 DEBUG [WriteFlusher] update WriteFlusher@40f03b3b{IDLE}:FAILED-->IDLE
09:54:44,328 DEBUG [HTTP2Session] Shutting down HTTP2ClientSession@292867a1{l:/10.110.254.4:60256 <-> r:/10.110.14.22:7443,queueSize=0,sendWindow=65535,recvWindow=65535,streams=0,NOT_CLOSED}
09:54:44,328 DEBUG [HTTP2Flusher] Failing, active/queued=2/0
09:54:44,328 DEBUG [HTTP2Session] Abrupt close for HTTP2ClientSession@292867a1{l:/10.110.254.4:60256 <-> r:/10.110.14.22:7443,queueSize=0,sendWindow=65535,recvWindow=65535,streams=0,NOT_CLOSED}
09:54:44,329 DEBUG [HTTP2Flusher] Terminated
09:54:44,329 DEBUG [HTTP2Session] Disconnecting HTTP2ClientSession@292867a1{l:/10.110.254.4:60256 <-> r:/10.110.14.22:7443,queueSize=0,sendWindow=65535,recvWindow=65535,streams=0,CLOSED}
09:54:44,330 DEBUG [SslConnection] SslConnection@531b92c7{NEED_UNWRAP,eio=-1/-1,di=-1} -> HTTP2ClientConnection@7213547c[DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,CLOSED,in,OSHUT,-,-,262/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,CLOSED,ISHUT,OSHUT,-,-,23/0,SslConnection@531b92c7}{io=1/1,kio=-1,kro=-1}] shutdownOutput: oshut=true, ishut=false
09:54:44,331 DEBUG [HTTP2Session] Ignoring close 0/close, already closed
09:54:44,331 DEBUG [WriteFlusher] ignored: WriteFlusher@40f03b3b{IDLE} java.nio.channels.ClosedChannelException
09:54:44,331 DEBUG [ExecuteProduceConsume] org.eclipse.jetty.http2.HTTP2Connection$HTTP2Producer@3f5c3f1 produced null
09:54:44,332 DEBUG [SslConnection] onFillable exit DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,CLOSED,in,OSHUT,-,-,264/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,CLOSED,ISHUT,OSHUT,-,-,25/0,SslConnection@531b92c7}{io=1/1,kio=-1,kro=-1}
09:54:44,333 DEBUG [SelectChannelEndPoint] task null
09:54:44,332 DEBUG [HttpExchange] Failed HttpExchange@40011e04 req=COMPLETED/org.eclipse.jetty.io.EofException@5ff9c652 res=COMPLETED/org.eclipse.jetty.io.EofException@5ff9c652: req=true/rsp=true org.eclipse.jetty.io.EofException
09:54:44,333 DEBUG [ExecuteProduceConsume] org.eclipse.jetty.io.ManagedSelector$SelectorProducer@bcf8bf produced org.eclipse.jetty.io.ManagedSelector$2@be8a41f
09:54:44,333 DEBUG [HttpExchange] Aborting while queued HttpExchange@40011e04 req=COMPLETED/org.eclipse.jetty.io.EofException@5ff9c652 res=COMPLETED/org.eclipse.jetty.io.EofException@5ff9c652: org.eclipse.jetty.io.EofException
09:54:44,334 DEBUG [QueuedThreadPool] queue org.eclipse.jetty.io.ManagedSelector$2@be8a41f
09:54:44,334 DEBUG [QueuedThreadPool] run org.eclipse.jetty.io.ManagedSelector$2@be8a41f
09:54:44,334 DEBUG [SelectChannelEndPoint] Key interests updated 1 -> 1 on SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,CLOSED,ISHUT,OSHUT,-,-,27/0,SslConnection@531b92c7}{io=1/1,kio=-1,kro=-1}
09:54:44,335 DEBUG [ManagedSelector] Destroyed SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,CLOSED,ISHUT,OSHUT,-,-,27/0,SslConnection@531b92c7}{io=1/1,kio=-1,kro=-1}
09:54:44,335 DEBUG [ManagedSelector] Selector loop waiting on select
09:54:44,335 DEBUG [QueuedThreadPool] ran org.eclipse.jetty.io.ssl.SslConnection$1@60ae489d
09:54:44,335 ERROR [ApnsManagedConnection] Error during APNS push notification:org.eclipse.jetty.io.EofException
java.util.concurrent.ExecutionException: org.eclipse.jetty.io.EofException
	at org.eclipse.jetty.client.util.FutureResponseListener.getResult(FutureResponseListener.java:118)
	at org.eclipse.jetty.client.util.FutureResponseListener.get(FutureResponseListener.java:110)
	at org.eclipse.jetty.client.HttpRequest.send(HttpRequest.java:654)
	at com.alu.udm.rcs.apns.ApnsClient.push(ApnsClient.java:101)
	at com.alu.udm.rcs.apns.AppleAPNSHttp2PushTest.pushMessage(AppleAPNSHttp2PushTest.java:65)
	at com.alu.udm.rcs.apns.AppleAPNSHttp2PushTest.testHttp2Connection(AppleAPNSHttp2PushTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.intellij.junit4.JUnit4TestRunnerUtil$IgnoreIgnoredTestJUnit4ClassRunner.runChild(JUnit4TestRunnerUtil.java:341)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
Caused by: org.eclipse.jetty.io.EofException
	at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.flush(SslConnection.java:786)
	at org.eclipse.jetty.io.WriteFlusher.flush(WriteFlusher.java:420)
	at org.eclipse.jetty.io.WriteFlusher.completeWrite(WriteFlusher.java:375)
	at org.eclipse.jetty.io.ssl.SslConnection$1.run(SslConnection.java:99)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:671)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:589)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.nio.channels.ClosedChannelException
	... 7 more
09:54:44,335 DEBUG [HTTP2Connection] HTTP2 Close HTTP2ClientConnection@7213547c[DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,CLOSED,in,OSHUT,-,-,267/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,CLOSED,ISHUT,OSHUT,-,-,28/0,SslConnection@531b92c7}{io=1/1,kio=-1,kro=-1}] 
java.net.ConnectException: org.eclipse.jetty.io.EofException
	at com.alu.udm.rcs.apns.ApnsClient.push(ApnsClient.java:112)
	at com.alu.udm.rcs.apns.AppleAPNSHttp2PushTest.pushMessage(AppleAPNSHttp2PushTest.java:65)
	at com.alu.udm.rcs.apns.AppleAPNSHttp2PushTest.testHttp2Connection(AppleAPNSHttp2PushTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.intellij.junit4.JUnit4TestRunnerUtil$IgnoreIgnoredTestJUnit4ClassRunner.runChild(JUnit4TestRunnerUtil.java:341)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:78)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:212)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
09:54:44,342 DEBUG [AbstractConnection] onClose HTTP2ClientConnection@7213547c[DecryptedEndPoint@f6f0a9c{/10.110.14.22:7443<->60256,CLOSED,in,OSHUT,-,-,274/0,HTTP2ClientConnection@7213547c}->SelectChannelEndPoint@1f37bb93{/10.110.14.22:7443<->60256,CLOSED,ISHUT,OSHUT,-,-,35/0,SslConnection@531b92c7}{io=1/1,kio=-1,kro=-1}]
i = 0
```

**Comment by sbordet** [comment]
Can you please try 9.3.16.v20170120 ? The fix for #1252 should have fixed this issue too.

**Comment by janssk1** [comment]
You are right. Works on 9.3.16
I should have tried the latest version..
